tests/fixture-data/txsource/staking-genesis.json

```diff
@@ -0,0 +1,15 @@
+{
+  "total_supply": "1000",
+  "ledger": {
+    "qihOAOzylETIN28lq+XmtKSFmTiEo3NKfHXHnjrV0rA=": {"general": {"balance": "100"}},
```

Comment:
Where did these accounts come from? Is this just what you get with the default seed?

Reply:
yeah it's that